dnd-troops-web/src/main/webapp/WEB-INF/jsp/troop/all.jsp

```diff
@@ -0,0 +1,64 @@
+<%@ page contentType="text/html;charset=UTF-8" pageEncoding="utf-8" trimDirectiveWhitespaces="true" session="false" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="my" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+
+<fmt:message var="title" key="troop.all.title"/>
+<my:admintemplate title="${title}">
+    <jsp:attribute name="body">
+        <h1><fmt:message key="troop.all.title"/></h1>
+
+        <form:form method="post" action="${pageContext.request.contextPath}/troop/search/name" modelAttribute="search" id="searchTroopName">
```

Comment:
User should be able to:
Find by name
Find by mission

Reply:
![image](https://user-images.githubusercontent.com/38562381/49827090-c65a8200-fd88-11e8-9ed4-019172ceda19.png)

which one is right?

Reply:
Both, there should be two different ways of searching for troop